test/Telegram.Bot.Tests.Integ/Exceptions/ApiExceptionsTests.cs

```diff
@@ -88,6 +88,27 @@ public ApiExceptionsTests(TestsFixture fixture)
             Assert.IsType<ContactRequestException>(e);
         }
 
+        [Fact(DisplayName = FactTitles.ShouldThrowExceptionMessageIsNotModifiedException)]
+        [Trait(Constants.MethodTraitName, Constants.TelegramBotApiMethods.SendMessage)]
+        [ExecutionOrder(5)]
+        public async Task Should_Throw_Exception_MessageIsNotModifiedException()
+        {
+            await _fixture.SendTestCaseNotificationAsync(FactTitles.ShouldThrowExceptionMessageIsNotModifiedException);
+
+            const string MessageTextToModify = "Message text to modify";
+            var message = await BotClient.SendTextMessageAsync(
```

Comment:
no `var` in tests. Write the type name. This helps people to reason about the code by just reading it.